openverse_catalog/dags/maintenance/check_silenced_dags/check_silenced_dags.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Tuple
+
+from airflow.exceptions import AirflowException
+from airflow.models import Variable
+from common.github import GitHubAPI
+from common.slack import send_alert
+
+
+logger = logging.getLogger(__name__)
+
+
+def check_configuration(github_pat: str, airflow_variable: str):
```

Comment:
Nit: Would you be comfortable reversing the order of these functions? I tend to read from top to bottom both when doing a review and accessing the code, and so having the order of composition match the order in the file can be really helpful for that. I know this repo isn't consistent about this, but it would be nice to do that ordering for new modules!